### PR TITLE
chore(META): format with 'sui-lint format-js ./components'

### PR DESCRIPTION
**Pull request by davidbarna**
Test to see how `sui-lint format-js` formats files

### CODE REVIEW

**Review by carlosvillu — APPROVED** [review]

File: components/ad/smartbanner/src/index.js [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
+  _handleClose = event => {
     const {onClose} = this.props
     event.preventDefault()
     onClose()
   }
 
   render () {
-    const { imageUrl, title, text, buttonText, staticPosition, icon: IconClose, ratingValue, ratingMax, customRatingIcons } = this.props
+    const {
+      imageUrl,
```
Comment (jcger) [inline components/ad/smartbanner/src/index.js:22]: could it be possible to sort the props?

File: components/atom/badgeCounter/src/index.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+import React, {Component} from 'react'
 
 class BadgeCounter extends Component {
   render () {
     return (
       <div className='sui-BadgeCounter'>
-        <span className={`sui-BadgeCounter--${this.props.size} sui-BadgeCounter-${this.props.type} sui-BadgeCounter-${this.props.type}--${this.props.thickness}`} />
+        <span
+          className={`sui-BadgeCounter--${this.props.size} sui-BadgeCounter-${
+            this.props.type
```
Comment (jcger) [inline components/atom/badgeCounter/src/index.js:10]: this looks kind of strange but maybe it should be implemented otherwise

**Review by jcger — COMMENTED** [review]
Hoy do you plan to release the packages? You did only one commit, not sure if the check:release script will be able to separate the releases

File: components/ad/smartbanner/src/index.js (hunk 1 above)
Comment (davidbarna) [reply components/ad/smartbanner/src/index.js:22]: nope

File: components/abtesting/optimizelyXExperiment/src/index.js [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
 
 class AbTestOptimizelyXExperiment extends Component {
-  state = { variation: undefined }
+  state = {variation: undefined}
 
   componentDidMount () {
     this._experiment = createExperimentUseCase(this.props.experimentId)
-    this._experiment.execute()
-      .then(variation => this.setState({variation}))
+    this._experiment.execute().then(variation => this.setState({variation}))
```
[the remark this answers is not in the document]
Comment (davidbarna) [reply components/abtesting/optimizelyXExperiment/src/index.js:13]: the line 80 limit.

File: components/abtesting/optimizelyXExperiment/src/index.js [hunk 4]
```diff
@@ -1,21 +1,24 @@
 /* eslint handle-callback-err: "off" */
 import PropTypes from 'prop-types'
 
-import React, { Component } from 'react'
+import React, {Component} from 'react'
```
[the remark this answers is not in the document]
Comment (davidbarna) [reply components/abtesting/optimizelyXExperiment/src/index.js:4]: Prettier has config with booleans, no options

File: components/atom/badgeCounter/src/index.js (hunk 2 above)
Comment (davidbarna) [reply components/atom/badgeCounter/src/index.js:10]: line limit again.

**Pull request closed without merging**